calculators/handlers/RodParts.js

```diff
@@ -28,6 +28,7 @@ class RodPartsHandler {
                     type: 'ROD_PART',
                     price: (prices[item.extraAttributes[type].part.toUpperCase()] ?? 0) * APPLICATION_WORTH.rodPart,
                     count: 1,
+                    soulbound: true,
```

Comment:
Should probably not include the calculation part if ignoring soulbound items.

Wait, doesn't SkyBlockItemCalculator have an `unsoulbound` property that says if the calculation includes soulbound? That would make all of this super easy by just wrapping this in an if block?